webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -17,18 +18,24 @@ const STATE_COLOR_MAP = {
     OPENED_COLOR: '#28a745',
     CLOSED_COLOR: '#cb2431',
     MERGED_COLOR: '#6f42c1',
+    ISSUE_CLOSED_COLOR: '#0b5cad',
 };
 
 const STATE_TYPES = {
     OPENED: 'opened',
     CLOSED: 'closed',
+    MERGED: 'merged',
 };
 
 const LINK_TYPES = {
     MERGE_REQUESTS: 'merge_requests',
     ISSUES: 'issues',
 };
 
+const ICON_WIDTH = 16;
```

Comment:
mention the icon name as well in the constant if its not for all icons like `TOOLTIP_ICON_WIDTH`